custom_components/circadian_lighting/switch.py

```diff
@@ -195,7 +200,7 @@ def turn_on(self, **kwargs):
         self._state = True
 
         # Make initial update
-        self.update_switch(DEFAULT_INITIAL_TRANSITION)
+        self.update_switch(CONF_INITIAL_TRANSITION)
```

Comment:
Can't do this. CONF_INITIAL_TRANSITION isn't an int, it's a configuration object. This _could_ be config.get(CONF_INITIAL_TRANSITION) but _should_ be:
```suggestion
        self.update_switch(self._initial_transition)
```